packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -93,19 +187,36 @@ export function SquadPageHeader({
           {squad.description}
         </p>
       )}
-      <SquadMemberShortList
-        squad={squad}
-        members={members}
-        className={classNames('my-6', !shouldUseListModeV1 && 'laptopL:hidden')}
-      />
+      <span className="mt-6 text-text-quaternary typo-footnote">
+        Moderated by
+      </span>
+      <div className="mt-2 flex flex-row items-center gap-3">
+        {squad.privilegedMembers?.slice(0, listMax).map((member) => (
```

Comment:
Why does this happen inline, can we abstract it outside the rendering?

Reply:
Not sure why not. Seems like a pretty common thing on the render function (similar to rendering infinite query results).

Reply:
Guess it was mostly annoying because this file is already so big and thought it would be nicer outside rendering, but really not urgent/needed

Reply:
Ah, I see. Yeah, maybe once we break it down, we might be able to find some way to make more bearable.